docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -11,1087 +89,1155 @@ blocks defined as dependencies. This abuses mechanics of the selector
+_Parameters_
 
- * state: Editor state.
- * clientId: Block client ID.
+-   _state_ `Object`: Editor state.
+-   _rootClientId_ `?string`: Optional block root client ID.
 
-*Returns*
+_Returns_
 
-Whether an ancestor of the block is in multi-selection
-                  set.
+-   `?string`: Block Template Lock
 
-### getMultiSelectedBlocksStartClientId
+<a name="hasInserterItems" href="#hasInserterItems">#</a> **hasInserterItems**
 
-Returns the client ID of the block which begins the multi-selection set, or
-null if there is no multi-selection.
+Determines whether there are items to show in the inserter.
 
-This is not necessarily the first client ID in the selection.
+_Parameters_
 
-*Parameters*
+-   _state_ `Object`: Editor state.
+-   _rootClientId_ `?string`: Optional root client ID of block list.
 
- * state: Editor state.
+_Returns_
 
-*Returns*
+-   `boolean`: Items that appear in inserter.
 
-Client ID of block beginning multi-selection.
+<a name="hasMultiSelection" href="#hasMultiSelection">#</a> **hasMultiSelection**
 
-### getMultiSelectedBlocksEndClientId
+Returns true if a multi-selection has been made, or false otherwise.
 
-Returns the client ID of the block which ends the multi-selection set, or
-null if there is no multi-selection.
+_Parameters_
 
-This is not necessarily the last client ID in the selection.
+-   _state_ `Object`: Editor state.
 
-*Parameters*
+_Returns_
 
- * state: Editor state.
+-   `boolean`: Whether multi-selection has been made.
 
-*Returns*
+<a name="hasSelectedBlock" href="#hasSelectedBlock">#</a> **hasSelectedBlock**
 
-Client ID of block ending multi-selection.
+Returns true if there is a single selected block, or false otherwise.
 
-### getBlockOrder
+_Parameters_
 
-Returns an array containing all block client IDs in the editor in the order
-they appear. Optionally accepts a root client ID of the block list for which
-the order should be returned, defaulting to the top-level block order.
+-   _state_ `Object`: Editor state.
 
-*Parameters*
+_Returns_
 
- * state: Editor state.
- * rootClientId: Optional root client ID of block list.
+-   `boolean`: Whether a single block is selected.
 
-*Returns*
+<a name="hasSelectedInnerBlock" href="#hasSelectedInnerBlock">#</a> **hasSelectedInnerBlock**
 
-Ordered client IDs of editor blocks.
+Returns true if one of the block's inner blocks is selected.
 
-### getBlockIndex
+_Parameters_
 
-Returns the index at which the block corresponding to the specified client
-ID occurs within the block order, or `-1` if the block does not exist.
+-   _state_ `Object`: Editor state.
+-   _clientId_ `string`: Block client ID.
+-   _deep_ `boolean`: Perform a deep check.
 
-*Parameters*
+_Returns_
 
- * state: Editor state.
- * clientId: Block client ID.
- * rootClientId: Optional root client ID of block list.
+-   `boolean`: Whether the block as an inner block selected
 
-*Returns*
+<a name="INSERTER_UTILITY_HIGH" href="#INSERTER_UTILITY_HIGH">#</a> **INSERTER_UTILITY_HIGH**
```

Comment:
These symbols shouldn't be documented. This will be fixed by https://github.com/WordPress/gutenberg/pull/15423 to be merged before this PR.

Reply:
15423 has been merged.